inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/fused_conv_eltwise_gpu_imad.cl

```diff
@@ -170,13 +287,41 @@ KERNEL (fused_convolution_eltwise_gpu_imad)(
             __attribute__((opencl_unroll_hint(FILTER_SIZE_X)))
             for (int kc = 0; kc < FILTER_SIZE_X; ++kc) // kc = Kernel Column
             {
+                #ifdef SHOULD_USE_DATA_ZP
+                    ACCUMULATOR_TYPE dotProdAZPxW = 0;
+                    dotProdAZPxW = TO_ACCUMULATOR_TYPE(
+                    IMAD(dotProdAZPxW,
+                    AS_INPUT0_TYPE_4(data_zp_val),
+                    AS_FILTER_TYPE_4(w[wi])));
+                #endif
+
                 __attribute__((opencl_unroll_hint))
                 for (int br = 0; br < OUT_BLOCK_HEIGHT; br++) {
                     __attribute__((opencl_unroll_hint))
                     for (int bc = 0; bc < OUT_BLOCK_WIDTH; bc++) {
-                        PACKED_TYPE input = sub_group_broadcast(in[br * STRIDE_SIZE_Y + kr * DILATION_SIZE_Y], bc * STRIDE_SIZE_X + kc * DILATION_SIZE_X);
+                        INPUT0_TYPE_4 inputs = AS_INPUT0_TYPE_4(sub_group_broadcast(in[br * STRIDE_SIZE_Y + kr * DILATION_SIZE_Y],
+                                                                                    bc * STRIDE_SIZE_X + kc * DILATION_SIZE_X));
 
-                        out[br * OUT_BLOCK_WIDTH + bc] = TO_ACCUMULATOR_TYPE(IMAD(out[br * OUT_BLOCK_WIDTH + bc], AS_INPUT0_TYPE_4(input), AS_FILTER_TYPE_4(w[wi])));
+                        out[br * OUT_BLOCK_WIDTH + bc] = TO_ACCUMULATOR_TYPE(IMAD(out[br * OUT_BLOCK_WIDTH + bc], inputs, AS_FILTER_TYPE_4(w[wi])));
+
+                        #ifdef ASYMMETRIC_WEIGHTS_QUANTIZATION
+                            ACCUMULATOR_TYPE dotProdAxWZP = 0;
+                            dotProdAxWZP = TO_ACCUMULATOR_TYPE(
```

Comment:
I think that you should use a single code style for IMAD (don't split code line as you did it on the line 305).